packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -3353,6 +3353,25 @@ describe('StripeHelper', () => {
       assert.deepEqual(result, customer1);
     });
 
+    it('fetches a customer and refreshes the cache if needed', async () => {
+      const customer = deepCopy(customer1);
+      customer.currency = null;
+      const customerSecond = deepCopy(customer1);
+      const expandStub = sandbox.stub(stripeHelper, 'expandResource');
+      stripeHelper.stripeFirestore = {
+        fetchAndInsertCustomer: sandbox.stub().resolves({}),
+      };
+      expandStub.onFirstCall().resolves(customer);
+      expandStub.onSecondCall().resolves(customerSecond);
+      const result = await stripeHelper.fetchCustomer(existingCustomer.uid);
+      assert.deepEqual(result, customer1);
```

Comment:
For clarity, should we assert `result` deep equals `customerSecond`? And maybe we can rename `customerSecond` to `customerWithCurrency`?

Reply:
Sure. 👍